dist/spec.yaml

```diff
@@ -4896,6 +4896,46 @@ paths:
       summary: Update SMS Template
       tags:
         - Template
+  /api/v1/threats/configuration:
+    get:
+      consumes:
+        - application/json
+      description: Gets current ThreatInsight configuration
+      operationId: getThreatInsightConfiguration
+      parameters: []
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/ThreatInsightConfiguration'
+      security:
+        - api_token: []
+      tags:
+        - ThreatInsight
+    post:
+      consumes:
+        - application/json
+      description: Updates ThreatInsight configuration
+      operationId: updateThreatInsightConfiguration
```

Comment:
If the client is already named `ThreatInsight` then we might want to rename this method to `updateConfiguration`, this way we'll have this: `ThreatInsightClient.UpdateConfiguration()`.